public/js/components/ArticleCommissionedLengthTable.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {Table, Tr, Th, Td} from './Table';
+
+const compare = (article1, article2) => article2.commissionedWordCount - article1.commissionedWordCount;
+
+const ArticleCommissionedLengthTable = ({articles}) => {
+    const noCommissioningLength = articles.filter(article => article.commissionedWordCount == 0)
+    const sortedArticles = articles.filter(article => article.commissionedWordCount > 0).sort(compare)
+    return(
+        <div>
+            <h3>Articles without a Commissioned Length</h3>
+            <Table alternate head={
+                <Tr>
+                    <Th>Headline</Th>
+                    <Th></Th>
+                    <Th>Word Count</Th>
+                </Tr>
+            }>
+                {noCommissioningLength.map(x => 
+                    <Tr key={x.path}>
+                        <Td><a href={"https://www.theguardian.com/"+x.path} target="_blank" rel="noopener noreferrer">{x.headline}</a></Td>
```

Comment:
For consistency I might usually use `https://www.theguardian.com/${x.path}` for string interpolation / concatenation these days.

Reply:
👍 fixed